src/map/script.c

```diff
@@ -15708,6 +15708,66 @@ BUILDIN(md5)
 	return true;
 }
 
+BUILDIN(swap)
+{
+	TBL_PC *sd = NULL;
+	struct script_data *data1, *data2;
+	const char *varname1, *varname2;
+	int64 uid1, uid2;
+
+	data1 = script_getdata(st,2);
+	data2 = script_getdata(st,3);
+
+	if (!data_isreference(data1) || !data_isreference(data2)) {
```

Comment:
after I tested with `report_data` script command,
`data_isreference` actually return 0 on both `reference_getconstant` or `data_isint`
which is, yeah, useless

but I'll add it just for additional check, no harm